test/utils.js

```diff
@@ -201,3 +201,36 @@ module.exports.closePanel = async(driver) => {
   const urlbar = await module.exports.promiseUrlBar(driver);
   await urlbar.sendKeys(webdriver.Key.ESCAPE);
 };
+
+// Returns array of pings of type `type` in sorted order by timestamp
+// first element is most recent ping
+// as seen in shield-study-addon-util's `utils.jsm`
+module.exports.getMostRecentPingsByType = async(driver, type) =>
+  driver.executeAsyncScript(async() => {
+    const typeArg = arguments[0];
+    const callback = arguments[arguments.length - 1];
+
+    Components.utils.import("resource://gre/modules/TelemetryArchive.jsm");
+    const pings = await TelemetryArchive.promiseArchivedPingList();
+
+    const filteredPings = pings.filter(p => p.type === typeArg);
+    filteredPings.sort((a, b) => {
+      if (a.timestampCreated > b.timestampCreated) {
+        return -1;
+      }
+      if (a.timestampCreated < b.timestampCreated) {
+        return 1;
+      }
+      if (a.timestampCreated === b.timestampCreated) {
+        return 0;
+      }
+      return 0;
```

Comment:
The comparator function doesn't care about `-1` and `1` specifically, it just wants positive and negative numbers. As such, you can simplify this and do something  like `filteredPings.sort((a, b) => a.timeStampCreated - b.timeStampCreated)`.